package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "@koale/useworker",
   "version": "2.0.6",
+  "types": "./index.d.ts",
   "source": "src/index.js",
   "main": "dist/index.js",
   "umd:main": "dist/index.umd.js",
   "module": "dist/index.mjs",
   "scripts": {
-    "build": "microbundle src/index.js && npm run copy",
+    "build": "microbundle src/index.js",
     "dev": "microbundle watch",
-    "copy": "cp package.json dist/ && cp README.md dist/",
```

Comment:
I would restore this and make two changes:
- Use a cross platform copy command (this won't work on Windows) like [cpx](https://www.npmjs.com/package/cpx)
- Copy the `index.d.ts` file too

Reply:
but would that be redundant? I already keep these files inside the root

Reply:
Sounds ok for the `index.d.ts` file. What about the `cpx` package?